package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-policy-watcher",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
In order to match with https://github.com/microsoft/vscode-policy-watcher/blob/main/package-lock.json#L3